src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -32,6 +32,7 @@
 
     // Independent Ui parts residing in this Ui container
```

Comment:
Since we seem to be standardising to capitalise UI.
```suggestion
    // Independent UI parts residing in this UI container
```